test/IRGen/asmname.swift

```diff
@@ -1,9 +1,50 @@
-// RUN: %target-swift-frontend -assume-parsing-unqualified-ownership-sil %s -emit-ir | %FileCheck %s
+// RUN: %target-swift-frontend -assume-parsing-unqualified-ownership-sil %s -emit-ir -whole-module-optimization | %FileCheck %s
```

Comment:
`-whole-module-optimization` isn't a frontend option.

Reply:
Unless I made a mistake I definitely saw -whole-module-optimization changing the output of this test, and I do see other tests that use this combination. I'll look again.

Reply:
Nope, I must have made a mistake somewhere. I see no output difference now.

Reply:
https://github.com/apple/swift/pull/12714